composer.json

```diff
@@ -4,20 +4,19 @@
     "type": "sulu-bundle",
     "license": "MIT",
     "require": {
-        "php": "^5.5 || ^7.0",
-        "sulu/sulu": "^1.5",
+        "php": "^7.1",
+        "sulu/sulu": "^2.0",
```

Comment:
sometimes this did make some problems when somebody requires `"sulu/sulu": "dev-develop"` so we should go here for:

```
    "sulu/sulu": "^2.0 || dev-develop"
```